apps/condo/domains/acquiring/access/Payment.js

```diff
@@ -61,10 +65,13 @@ async function canManagePayments ({ authentication: { item: user }, operation, i
     return false
 }
 
-async function canReadPaymentsSensitiveData ({ authentication: { item: user }, existingItem, context }) {
+async function canReadPaymentsSensitiveData ({ authentication: { item: user }, existingItem, context, listKey }) {
     if (!user || user.deletedAt) return false
     if (user.isSupport || user.isAdmin) return true
 
+    const hasDirectAccess = await canDirectlyReadSchemaObjects(user, listKey)
+    if (hasDirectAccess) return true
+
```

Comment:
Is it ok that we give direct access to sensitive data?

Reply:
I do not really see caveats here, if you have direct access for list you should be, imo, able to see all fields

Reply:
I compared the accesses, they seem similar, so ok